src/Apps/Artist/Routes/Overview/state.ts

```diff
@@ -94,9 +94,7 @@ export class FilterState extends Container<State> {
     if (props) {
       Object.keys(this.state).forEach(filter => {
         if (props[filter]) {
-          if (filter === "major_periods") {
```

Comment:
Gotcha, so these changes cause the `major_periods` case to fall through to the general `else` clause – nice!

Reply:
Oh, yes! I intended on pointing that out so you didn't have to figure it out, but I forgot.